docs/targets/logz.md

```diff
@@ -0,0 +1,47 @@
+# Event Target for Logz.io 
+This event target integrates with Logz.io, using received CloudEvent's to ship messages to [Logz.io](https://logz.io/)
+
+## Prerequisites
+* Logz.io account.
+* A Logz.io Shipping [token](https://docs.logz.io/user-guide/tokens/)
+
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **Shipping Token** The API Shipping Token. 
```

Comment:
Should we add more infor about what an API shipping token is. This seems redundant to have Shipping Token - Api Shipping token. I would either add what a shipping token is or just remove the redundancy